.rubocop.yml

```
@@ -126,3 +126,47 @@ Style/OptionalBooleanParameter:
   Enabled: false
 Style/RedundantRegexpEscape:
   Enabled: false
+Gemspec/RequireMFA: # new in 1.23
+  Enabled: true
+Layout/LineEndStringConcatenationIndentation: # new in 1.18
+  Enabled: true
+Lint/AmbiguousOperatorPrecedence: # new in 1.21
+  Enabled: true
+Lint/AmbiguousRange: # new in 1.19
+  Enabled: true
+Lint/EmptyInPattern: # new in 1.16
+  Enabled: true
+Lint/IncompatibleIoSelectWithFiberScheduler: # new in 1.21
+  Enabled: true
+Lint/RequireRelativeSelfPath: # new in 1.22
+  Enabled: true
+Lint/UselessRuby2Keywords: # new in 1.23
+  Enabled: true
+Naming/BlockForwarding: # new in 1.24
+  Enabled: true
```

Can we update this rule to the following? Or just disable it. Unless there are strong opinions otherwise I think it is easier to read a function that is passing `&block` instead of just `&`. It's updated it in a couple places with the autocorrect. I'll note the first one I come to.
```suggestion
   EnforcedStyle: explicit
```

---

Yeah I'll disable